oidc-client-app-samples/android-client-app-sample/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
Missing license header.

Reply:
Fixed in PR: https://github.com/wso2/samples-is/pull/179